### ISSUE TITLE
daily views

### ISSUE THREAD

**Issue by AltaiQaghan** [yt-dlp/yt-dlp#4988]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.09.01** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I'm trying to download selected video's from a channel on youtube. There is no option to filter video's on average daily views that i know of so I do it like this;

yt-dlp -i --progress -x --audio-format mp3 --audio-quality 0 
--match-filter "view_count>?1000000 & upload_date>?20220821" 
--match-filter "view_count>?2000000 & upload_date>?20220721" 
--match-filter "view_count>?3000000 & upload_date>?20220621" 
--match-filter "view_count>?4000000 & upload_date>?20220521" 
--match-filter "view_count>?5000000 & upload_date>?20220421" 
--match-filter "view_count>?6000000 & upload_date>?20220321" 
 -o "%(uploader)s/%(playlist)s/%(title)s.%(ext)s" https://www.youtube.com/c/netdmuzik/videos

(There are no new lines when i paste this but just now for readability.) 
This way i download the most popular videos's from the last 6 months with a minimum view count of one million a month. 

I want to ask if there is way i dont have to change the dates every time but use something like 'today-1month'.
This is possible with the --dateafter but that way i cant have multiple filters.

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by pukkandan** [comment]
It's not currently possible

**Comment by AltaiQaghan** [comment]
So I understand it's not possible with upload_date and also not with the timestamp filters. Can I make a request here for adding a video filter like '--match_filters daily_view_count' ?

**Comment by AltaiQaghan** [comment]
Dear pukkandan, thank you for your interest in this so far. I don't understand the changed title. I wanted to download videos for example maximum a year old, but the main question was: Can i filter videos on **average daily views**

If there is a misunderstanding could you please reopen my request?

**Comment by pukkandan** [comment]
> Can i filter videos on **average daily views**

No, and I don't think it makes any sense to add such a new field since it's not something the site directly provides.

That is what you ultimately want to achieve, but at the end, the request was this, no?

> I want to ask if there is way i dont have to change the dates every time but use something like 'today-1month'.
This is possible with the --dateafter but that way i cant have multiple filters.
